config/routes.rb

```diff
@@ -53,19 +53,22 @@
   #     # (app/controllers/admin/products_controller.rb)
   #     resources :products
   #   end
+  concern :paginatable do
```

Comment:
<code>pagination</code> does not follow the convention of rails routes concern ><

I learnt the paginatable name from kaminari's documentation: https://github.com/amatsuda/kaminari#creating-friendly-urls-and-caching